### PR TITLE
GLSL version dependent on OpenGL version.

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

File: gapic/src/main/com/google/gapid/glviewer/ShaderSource.java [hunk 1]
```diff
@@ -37,16 +42,18 @@ public static Shader loadShader(String name) {
   }
 
   public static Shader loadShader(URL resource) {
+    String version = (GL11.glGetInteger(GL30.GL_MAJOR_VERSION) == 3 &&
```
Comment (ben-clayton) [inline gapic/src/main/com/google/gapid/glviewer/ShaderSource.java]: The test on GL_MAJOR_VERSION here seems iffy. What are the possible values? Maybe add a versionAtLeast function?

Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/glviewer/ShaderSource.java]: Done.

**Comment by Qining** [comment]
Confirmed this works on my laptop (Ubuntu 1404) now. LGTM

**Comment by ben-clayton** [comment]
Thanks

**Pull request merged**